lib/remote-ftp.js

```diff
@@ -319,6 +319,31 @@ module.exports = {
 					});
 				});
 			},
+			
+			'remote-ftp:download-tab': function ( e ) {
+				if (!hasProject()) return;
+
+				atom.project.remoteftp.connect();
```

Comment:
Please don't reconnect every time...
The `upload` and `download` functions` use the `self.onceConnected(callback)` function which ensures there is a connection (and connects if necessary).